fix issues in the desktop file (no icon, hardcodes xterm)

upstreamed in wb2osz/direwolf#393

--- a/cmake/cpack/direwolf.desktop.in
+++ b/cmake/cpack/direwolf.desktop.in
@@ -2,9 +2,9 @@
 Name=@APPLICATION_NAME@
 Comment=APRS Soundcard TNC
-Exec=@APPLICATION_DESKTOP_EXEC@
+Exec=@CMAKE_PROJECT_NAME@
-Icon=@CMAKE_PROJECT_NAME@_icon.png
+Icon=@CMAKE_PROJECT_NAME@_icon
 StartupNotify=true
-Terminal=false
+Terminal=true
 Type=Application
 Categories=HamRadio
 Keywords=Ham Radio;APRS;Soundcard TNC;KISS;AGWPE;AX.25
 
